presto/create.sql

```diff
@@ -0,0 +1,107 @@
+    NetMajor SMALLINT NOT NULL,
+    NetMinor SMALLINT NOT NULL,
+    UserAgentMajor SMALLINT NOT NULL,
+    UserAgentMinor VARCHAR(65535) NOT NULL,
+    CookieEnable SMALLINT NOT NULL,
+    JavascriptEnable SMALLINT NOT NULL,
+    IsMobile SMALLINT NOT NULL,
+    MobilePhone SMALLINT NOT NULL,
+    MobilePhoneModel VARCHAR(65535) NOT NULL,
+    Params VARCHAR(65535) NOT NULL,
+    IPNetworkID INT NOT NULL,
+    TraficSourceID SMALLINT NOT NULL,
+    SearchEngineID SMALLINT NOT NULL,
+    SearchPhrase VARCHAR(65535) NOT NULL,
+    AdvEngineID SMALLINT NOT NULL,
+    IsArtifical SMALLINT NOT NULL,
+    WindowClientWidth SMALLINT NOT NULL,
+    WindowClientHeight SMALLINT NOT NULL,
+    ClientTimeZone SMALLINT NOT NULL,
+    ClientEventTime TIMESTAMP NOT NULL,
+    SilverlightVersion1 SMALLINT NOT NULL,
+    SilverlightVersion2 SMALLINT NOT NULL,
+    SilverlightVersion3 INT NOT NULL,
+    SilverlightVersion4 SMALLINT NOT NULL,
+    PageCharset VARCHAR(65535) NOT NULL,
+    CodeVersion INT NOT NULL,
+    IsLink SMALLINT NOT NULL,
+    IsDownload SMALLINT NOT NULL,
+    IsNotBounce SMALLINT NOT NULL,
+    FUniqID BIGINT NOT NULL,
+    OriginalURL VARCHAR(65535) NOT NULL,
+    HID INT NOT NULL,
+    IsOldCounter SMALLINT NOT NULL,
+    IsEvent SMALLINT NOT NULL,
+    IsParameter SMALLINT NOT NULL,
+    DontCountHits SMALLINT NOT NULL,
+    WithHash SMALLINT NOT NULL,
+    HitColor VARCHAR(65535) NOT NULL,
+    LocalEventTime TIMESTAMP NOT NULL,
+    Age SMALLINT NOT NULL,
+    Sex SMALLINT NOT NULL,
+    Income SMALLINT NOT NULL,
+    Interests SMALLINT NOT NULL,
+    Robotness SMALLINT NOT NULL,
+    RemoteIP INT NOT NULL,
+    WindowName INT NOT NULL,
+    OpenerName INT NOT NULL,
+    HistoryLength SMALLINT NOT NULL,
+    BrowserLanguage VARCHAR(65535) NOT NULL,
+    BrowserCountry VARCHAR(65535) NOT NULL,
+    SocialNetwork VARCHAR(65535) NOT NULL,
+    SocialAction VARCHAR(65535) NOT NULL,
+    HTTPError SMALLINT NOT NULL,
+    SendTiming INT NOT NULL,
+    DNSTiming INT NOT NULL,
+    ConnectTiming INT NOT NULL,
+    ResponseStartTiming INT NOT NULL,
+    ResponseEndTiming INT NOT NULL,
+    FetchTiming INT NOT NULL,
+    SocialSourceNetworkID SMALLINT NOT NULL,
+    SocialSourcePage VARCHAR(65535) NOT NULL,
+    ParamPrice BIGINT NOT NULL,
+    ParamOrderID VARCHAR(65535) NOT NULL,
+    ParamCurrency VARCHAR(65535) NOT NULL,
+    ParamCurrencyID SMALLINT NOT NULL,
+    OpenstatServiceName VARCHAR(65535) NOT NULL,
+    OpenstatCampaignID VARCHAR(65535) NOT NULL,
+    OpenstatAdID VARCHAR(65535) NOT NULL,
+    OpenstatSourceID VARCHAR(65535) NOT NULL,
+    UTMSource VARCHAR(65535) NOT NULL,
+    UTMMedium VARCHAR(65535) NOT NULL,
+    UTMCampaign VARCHAR(65535) NOT NULL,
+    UTMContent VARCHAR(65535) NOT NULL,
+    UTMTerm VARCHAR(65535) NOT NULL,
+    FromTag VARCHAR(65535) NOT NULL,
+    HasGCLID SMALLINT NOT NULL,
+    RefererHash BIGINT NOT NULL,
+    URLHash BIGINT NOT NULL,
+    CLID INT NOT NULL
+);
```

Comment:
Would it be easier for the users if we add the WITH clause here?

Reply:
Fixed now.